vault/changelog.release.2022-02-22.md

```diff
@@ -0,0 +1,130 @@
+---
+id: ixhyxsx79zqjy25s7q9liud
+title: '0.83'
+desc: ''
+updated: 1645493812159
+created: 1645488383028
+---
+
+Dendron 0.83 has sprouted  🌱
+
+**Breaking changes:** This release of Dendron updates the publishing configuration within `dendron.yml`. This update is not backwards compatible, meaning that older versions of `dendron-cli` will not be able to publish a migrated workspace.
+
+- More information: [[0.83 Changelog|dendron://dendron.dendron-site/changelog.release.2022-02-22#changelog]].
+
+Pod V2 preview continues to grow: you can now export the contents of a hierarchy, filtered by a target vault. This means the multi-vault workspaces can be more targeted in their exports without including notes from undesired vaults.
+
+- More information: [[Pod v2 exportScope|dendron://dendron.dendron-site/dendron.topic.pod-v2.config#exportscope]]
+
+File Links now support block anchors (` ^block-anchor`)! This means that you can create comments in your code, and link directly to the comment blocks. Optionally, you can link line numbers.
```

Comment:
> Optionally, you can link line number

this was the old behavior. we're not introducing it - would change to mentioning that this behavior is configurable